### ISSUE TITLE
An optional `--help` contract should be established for every script

### ISSUE THREAD

**Issue by blaggacao** [fluidattacks/makes#547]
```console
$ m . /mintNativeToken/prepareWallet --help
Makes v21.10-linux

This job prepares a cardano wallet for minting native tokens.

...
```
```console
$ m . /mintNativeToken/someOther --help
Makes v21.10-linux

This job does not provide a help.

...
```

**Comment by blaggacao** [comment]
x-ref #521

**Comment by blaggacao** [comment]
This has been implemented via 047169c1ae54046c6c191cfbacf4c6100144e21d
